apps/financial-aid/web-veita/src/components/MunicipalityAdminSettings/MunicipalityAdminSettings.tsx

```diff
@@ -9,24 +8,61 @@ import {
   ToastContainer,
 } from '@island.is/island-ui/core'
 
-import { Municipality } from '@island.is/financial-aid/shared/lib'
+import {
+  Aid,
+  AidName,
+  Municipality,
+  scrollToId,
+} from '@island.is/financial-aid/shared/lib'
 import { useMutation } from '@apollo/client'
 import { UpdateMunicipalityMutation } from '@island.is/financial-aid-web/veita/graphql'
 import omit from 'lodash/omit'
 import { useMunicipality } from '@island.is/financial-aid/shared/components'
+import MunicipalityAdminInput from './MunicipalityNumberInput/MunicipalityNumberInput'
 
 interface Props {
   municipality: Municipality
 }
 
 const MunicipalityAdminSettings = ({ municipality }: Props) => {
-  const maxAmountLength = 6
   const [state, setState] = useState(municipality)
+  const [hasAidError, setHasAidError] = useState(false)
   const [updateMunicipalityMutation, { loading }] = useMutation(
     UpdateMunicipalityMutation,
   )
   const { setMunicipality } = useMunicipality()
 
+  const INDIVIDUAL = 'individual'
+  const COHABITATION = 'cohabitation'
+  const aidNames = Object.values(AidName).map(String)
+
+  const errorCheck = (aid: Aid, prefix: string) => {
+    for (const a of Object.entries(aid)) {
+      if (aidNames.includes(a[0]) && a[1] === 0) {
+        setHasAidError(() => true)
```

Comment:
why is this function since hasAidError is boolean, why not just setHasAidError(true)